pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>uk.gov.companieshouse</groupId>
+		<artifactId>companies-house-spring-boot-parent</artifactId>
+		<version>1.2.0</version>
+	</parent>
+	<artifactId>chd-order-consumer</artifactId>
+	<version>unversioned</version>
+	<name>chd-order-consumer</name>
+	<description>CH Service consuming kafka messages for Filing History Image Scan Requests</description>
+
+	<properties>
+		<java.version>1.8</java.version>
+		<org.mapstruct.version>1.3.0.Final</org.mapstruct.version>
+		<jacoco-maven-plugin.version>0.7.7.201606060606</jacoco-maven-plugin.version>
```

Comment:
If you use the latest version of parent pom , you can inherit the jacoco version from that.
https://github.com/companieshouse/parent-pom/blob/master/pom.xml

Reply:
updated